style.css

```diff
@@ -235,6 +250,17 @@
   height: auto;
 }
 
+.dec {
+  display: none;
+}
+
+.canopy {
+  font-weight: 500;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
 .card-content {
```

Comment:
- [ ] :thinking: As requested by the previous reviewer [here](https://github.com/QueenterJuma/Portfolio_one/pull/4/files#r1081959557), Kindly increase again the distance between the card title and the top to match the Figma design.